File: tbot-application/src/main/java/org/andreidodu/tbot/dto/DomandaDTO.java
package org.andreidodu.tbot.dto;

import java.util.ArrayList;
import java.util.List;

import lombok.Data;


@Data
public class DomandaDTO {
	
	private String id;
	private String codiceGruppoDomanda;
	private String formato;
	private String codiceDominio;
	private String testo;
	private String suggerimento;
	private Boolean obbligatorio;
	private Integer order;
	
	
	private String valore;
	
	private List<DominioDTO> dominio = new ArrayList<>();
	
}

File: tbot-web/src/main/tbot-web/src/app/modules/common/components/form-input/dto/domanda-dto.ts
import {DominioDto} from './dominio-dto';

export interface DomandaDto {

  id: string;
  codiceGruppoDomanda: string;
  formato: string;
  testo: string;
  dominio: Array<DominioDto>;
  valore: string;
  suggerimento: string;
  obbligatorio: boolean;

}
